packages/server-api/src/index.ts

```diff
@@ -172,6 +172,17 @@ export interface ServerAPI extends PluginServerApp {
     ) => void
   }) => void
   getSerialPorts: () => Promise<Ports>
+
+  getFeatures: () => {
+    apis: string[]
```

Comment:
I think the "well known APIs" should be present in the server API typings. Move SIGNALK_API_ID here?